### PR TITLE
GB12 general settings

### PR DESCRIPTION
**Pull request by kris-kolosek**

### CODE REVIEW

File: app/controllers/general_settings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class GeneralSettingsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_general_setting, only: %i(edit)
```
Comment (salac-kolosek) [inline app/controllers/general_settings_controller.rb]: Use :edit if only one method.

File: app/controllers/stories_controller.rb [hunk 2]
```diff
@@ -31,6 +31,7 @@ def update
       redirect_to story_path(@story)
     else
       flash[:error] = "Error while updating story"
+      redirect_to story_path(@story)
```
Comment (salac-kolosek) [inline app/controllers/stories_controller.rb]: Write in one line redirect_to ..., error: message.

File: config/routes.rb [hunk 3]
```diff
@@ -23,5 +23,10 @@
   get "team", to: "users#index", as: "team"
 
   get "site-overviews", to: "site_overviews#index", as: "site-overviews"
-  get "site-overview", to: "site_overviews#show", as: "site-overview"
+  get "site-overview/:id", to: "site_overviews#show", as: "site-overview"
```
Comment (salac-kolosek) [inline config/routes.rb]: Please remove these routes. You need to use resources instead of a resource in your routes config file to fix this problem.

Comment (kris-kolosek) [reply config/routes.rb]: I will fix it, thanks for explaining :)

**Pull request merged**